src/main/java/uk/gov/mint/MintService.java

```diff
@@ -21,6 +22,8 @@ public MintService(LoadHandler loadHandler) {
     @PermitAll
     @Path("/load")
     public void load(String payload) {
-        loadHandler.handle(payload);
+        for (Handler h : loadHandlers) {
```

Comment:
Okay I think I may have made a mistake writing the story. I know I said "write to the CT log server in parallel" but I'm not sure that it makes sense for `MintService` to have a list of `Handler`s, because having a list of handlers makes handling partial failure much harder. What if postgres succeeds but the CT log fails?

I wonder if it would be better to allow a mint to have either a postgres or a ct log backend, but not both.

Is there a way of doing this PR such that we configure